src/main/java/com/denys/toukdemo/service/MailServiceImpl.java

```diff
@@ -0,0 +1,35 @@
+package com.denys.toukdemo.service;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.mail.MailSender;
+import org.springframework.mail.SimpleMailMessage;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MailServiceImpl implements MailService{
+
+    private static final String CONFIRMATION_EMAIL = "toukdemo@yahoo.com";
+    private static final String CONFIRMATION_EMAIL_SUBJECT = "Touk demo booking confirmation";
+    private static final String CONFIRMATION_EMAIL_BODY = "Hi! Please confirm your booking by link! ";
+    private static final String CONFIRMATION_LINK = "http://localhost:8080/confirmation?no=";
```

Comment:
a jak pójdziemy z tym na produkcję to taki adres będzie ok?

Reply:
> a jak pójdziemy z tym na produkcję to taki adres będzie ok?

nie, nie ok
można dodać możliwość wczytania czegoś takiego z plików konfiguracyjnych